moveit_ros/planning/kinematics_plugin_loader/src/kinematics_parameters.yaml

```diff
@@ -0,0 +1,30 @@
+kinematics:
+  kinematics_solver: {
+    type: string,
+    default_value: kdl_kinematics_plugin/KDLKinematicsPlugin,
+    description: "Kinematics solver",
+    validation: {
+      not_empty<>: []
+    }
+  }
+  kinematics_solver_search_resolution: {
+    type: double,
+    default_value: 0.1,
+    description: "kinematics solver search resolution",
+    validation: {
+      lower_bounds<>: [ 0.0 ]
+    }
+  }
+  kinematics_solver_timeout: {
+    type: double,
+    default_value: 0.05,
+    description: "kinematics solver timeout",
+    validation: {
+      lower_bounds<>: [ 0.0 ]
```

Comment:
Should 0 be a possible value or should the lower bounds for this be something slightly larger than zero? Would it be better to change the names of these validators to be something like:

* gt
* gt_eq
* lt
* lt_eq

Then you could specify that this is greater than 0 (but not 0).

Reply:
https://github.com/PickNikRobotics/generate_parameter_library/pull/80/files